dash-chatbot-app/app.py

```diff
@@ -21,4 +20,4 @@
 ], fluid=True)
 
 if __name__ == '__main__':
-    app.run_server(debug=True)
+    app.run(debug=True)
```

Comment:
Thanks for fixing this! The template is actually broke in production right now because dash 3+ does not support this method anymore. 😅